molecule/accessible-namespaces-test/kiali-cr.yaml

```diff
@@ -13,4 +13,4 @@ spec:
     image_pull_policy: {{ kiali.image_pull_policy }}
     image_version: {{ kiali.image_version }}
     accessible_namespaces: {{ kiali.accessible_namespaces }}
-    service_type: NodePort
+    service_type: {{ 'LoadBalancer' if is_kind else 'NodePort' }}
```

Comment:
we have a new test "jaeger-test" that needs this same change: https://github.com/kiali/kiali-operator/blob/master/molecule/jaeger-test/kiali-cr.yaml#L16

Also token-test is soon going to get its own kiali-cr.yaml so that will need to be given this same change, too: https://github.com/kiali/kiali-operator/pull/366/files#diff-ef0348c316c4e1a05dfae679ef7e6d9c15d225c3c62ad0abaaf746b12c2f256dR16

Reply:
I integrated changes in master to do the needed changes.